src/util/DirectoryExportedClassesLoader.ts

```diff
@@ -6,16 +6,16 @@ import {PlatformTools} from "../platform/PlatformTools";
 export function importClassesFromDirectories(directories: string[], formats = [".js", ".ts"]): Function[] {
 
     function loadFileClasses(exported: any, allLoaded: Function[]) {
-        if (exported instanceof Function) {
+        if (typeof exported === "function") {
             allLoaded.push(exported);
 
-        } else if (exported instanceof Object) {
+        } else if (Array.isArray(exported)) {
```

Comment:
No you can left it as it is, however next time don't change exist code if its the same. Because for example `instanceof Array` used in 100 other places and when one change differs it brings questions and inconsistency